extensions/wikia/MercuryApi/models/MercuryApi.class.php

```diff
@@ -146,6 +146,9 @@ public function getWikiVariables() {
 			'theme' => SassUtil::normalizeThemeColors( SassUtil::getOasisSettings() ),
 			'tracking' => [
 				'vertical' => HubService::getVerticalNameForComscore( $wgCityId ),
+				'comscore' => [
+					'c7Value' => AnalyticsProviderComscore::getC7Value(),
```

Comment:
For safety: Disabled comscore tracking in mercury if `c7Value` is `false`. The same condition is implemented in: https://github.com/Wikia/app/blob/d676be1e51dd87a5a16bdc4948f09ac4d34ee3fc/extensions/wikia/AnalyticsEngine/AnalyticsProviderComscore.php#L15-L15